packages/celer-web-app/src/data/storage/RecentPages.ts

```diff
@@ -0,0 +1,41 @@
+import { LocalStorageWrapper } from ".";
+
+const KEY = "RecentlyVisitedPages";
+const NUM_PAGES = 5; // <-- Change this to increase number of pages tracked
+const DEFAULT = new Array(NUM_PAGES);
```

Comment:
Do you need array of 5? From what I see you are just filling empty strings and generating invisible DOM elements. Could you allow the array to be shorter than 5?

Reply:
This should be fixed with the most recent implementation. It is now a maximum size with shorter arrays being possible rather than always being length 5 and filling in empty values.

Reply:
That being said, do you want to change the maximum number of pages at all? I thought 5 seemed reasonable, but that's a quick fix if you like something else. Keep in mind, the link to the Dev Server will always appear at the bottom, so ``MAX_PAGES = 5`` effectively has a list with up to 6 elements.

Reply:
Let's keep this as a constant for now and potentially make it a setting in the settings dialog in the next steps